UnitsNet/UnitDefinitions/Speed.json

```diff
@@ -78,5 +78,6 @@
         }
       ]
     }
-  ]
+  ],
+  "SiArray": [ 1, 0, -1, 0, 0, 0, 0 ]
```

Comment:
I'm just trying to understand still, so you're adding the SI array to each quantity, which represents the base unit?
In the case of speed, it's `m/s`. And you use this to automatically generate all possible `*` and `/` operators across quantities, such as `Length / Time = Speed`. 

Neat!

Thinking further, can we use this to 
1. Convert to other speed units? Use `FromBaseToUnitFunc` or do we add an `SiArray` to each unit?

Reply:
I'm considering creating the SI-arrays by parsing the unit names (or abbreviations), but I'm not sure if that is actually faster than just creating the arrays by hand.

It would be possible to create a similar double array with conversion factors for other unit systems (like Imperial). That could be used to automatically generate the code for conversion to that unit system. Since we already have Imperial for most quantities I'm not sure if it is really necessary. But it would be nice when adding new quantities.